src/components/Loading/Loading.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Loading from './Loading';
+
+describe('Loading', () => {
+  it('renders the Loading component', () => {
+    render(<Loading />);
+
+    const loadingElement1 = screen.getByTestId('loading-1');
+    expect(loadingElement1).toBeInTheDocument();
+
+    const loadingElement2 = screen.getByTestId('loading-2');
```

Comment:
Render olan component'in içeriğine child, firstChild, haveContent gibi methodlarla erişebiliyoruz. Children'a ayrı test-id vermemize gerek yok.